### PR TITLE
Feature/28 controller unit tests

### LINKED ISSUE #28: Controller unit tests

### PR DESCRIPTION
**Pull request by gregdeane**
No controller spec file necessary. All controller tests performed in component spec file.

Closes #28

### CODE REVIEW

File: src/app/feedback.component.spec.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import ComponentTest from '../util/test.component';
 import ScFeedbackController from './feedback.controller';
 
 import IAugmentedJQuery = angular.IAugmentedJQuery;
+import IInjectorService = angular.auto.IInjectorService;
 
 describe('ScFeedbackComponent', () => {
-  let sut: ComponentTest<ScFeedbackController>;
-  let mock: any;
+  let sut: ComponentTest<ScFeedbackController>;    // sut = system under test
```
Comment (dami-gg) [inline src/app/feedback.component.spec.ts]: Thanks for the comment, I'd have never got its meaning by myself :D

**Comment by dami-gg** [comment]
Awesome work, all the tests pass and the new syntax is quite clean and easy to understand. Ready to merge!

**Comment by dami-gg** [comment]
:+1:

File: src/app/feedback.component.spec.ts (hunk 1 above)
Comment (gregdeane) [inline src/app/feedback.component.spec.ts]: Haha! It took me a while the first time I saw it in someone else's code. But it's nice if you don't want a long-winded var name for the component in question.

**Comment by gregdeane** [comment]
👍

**Pull request merged**